client/webpack.config.js

```diff
@@ -12,7 +12,9 @@ const config = {
 
   output: {
     filename: 'coursemology.bundle.js',
+    chunkFilename: '[name]-[chunkhash].bundle.js',
```

Comment:
It's like a special placeholder from webpack: https://webpack.github.io/docs/long-term-caching.html

they will be replaced with the hash of the file after compiling.